src/main/java/de/srendi/advancedperipherals/common/blocks/base/PeripheralBlockEntity.java

```diff
@@ -82,6 +87,12 @@ public <T1> LazyOptional<T1> getCapability(@NotNull Capability<T1> cap, @Nullabl
                 handler = LazyOptional.of(() -> new SidedInvWrapper(this, Direction.NORTH));
             return handler.cast();
         }
+
+        if (cap == CapabilityFluidHandler.FLUID_HANDLER_CAPABILITY && !remove && direction != null) {
+            if (fluidHandler == null || !fluidHandler.isPresent())
+                fluidHandler = LazyOptional.of(() -> new FluidTank(1000));
```

Comment:
Everything seems to work fine. But the capacity should be zero, since the bridge does not do anything with the contained fluid. The bridge would just store one bucket of liquid forever.